geode-core/src/main/java/org/apache/geode/management/internal/cli/domain/DataCommandResult.java

```diff
@@ -406,124 +400,140 @@ public Result toCommandResult() {
       section.addData("Message", errorString);
       section.addData(RESULT_FLAG, operationCompletedSuccessfully);
       return ResultBuilder.buildResult(data);
+    }
+
+    CompositeResultData data = ResultBuilder.createCompositeResultData();
+    SectionResultData section = data.addSection();
+    TabularResultData table = section.addTable();
+
+    section.addData(RESULT_FLAG, operationCompletedSuccessfully);
+    if (infoString != null) {
+      section.addData("Message", infoString);
+    }
+
+    if (isGet()) {
+      toCommandResult_isGet(section, table);
+    } else if (isLocateEntry()) {
+      toCommandResult_isLocate(section, table);
+    } else if (isPut()) {
+      toCommandResult_isPut(section, table);
+    } else if (isRemove()) {
+      toCommandResult_isRemove(section, table);
+    } else if (isSelect()) {
+      // its moved to its separate method
```

Comment:
so, for isSelect() we don't do anything? looks like isSelect() is only used here, probably we can remove this function? looks like there is a lot of unused functions in this class as well.